# HG changeset patch
# User Rik <rik@octave.org>
# Date 1373244938 25200
#      Sun Jul 07 17:55:38 2013 -0700
# Node ID 908b4956cbfddc856144be4968a287597f2a4ba0
# Parent  ab46b5c77669934edfe50cd02b23d93089a179b5
doc: remove '\b' from list of special regexp sequences (bug #38778).

* libinterp/corefcn/regexp.cc(Fregexp): remove '\b' from list of special regexp
sequences (bug #38778).

diff --git a/libinterp/corefcn/regexp.cc b/libinterp/corefcn/regexp.cc
--- a/libinterp/corefcn/regexp.cc
+++ b/libinterp/corefcn/regexp.cc
@@ -642,45 +642,43 @@ alternatives must be delimited by the gr
 @item ^ $\n\
 Anchoring operators.  Requires pattern to occur at the start (@code{^}) or\n\
 end (@code{$}) of the string.\n\
 @end table\n\
 \n\
 In addition, the following escaped characters have special meaning.\n\
 \n\
 @table @code\n\
-@item \\b\n\
-Match a word boundary\n\
+\n\
+@item \\d\n\
+Match any digit\n\
 \n\
-@item \\B\n\
-Match within a word\n\
+@item \\D\n\
+Match any non-digit\n\
+\n\
+@item \\s\n\
+Match any whitespace character\n\
+\n\
+@item \\S\n\
+Match any non-whitespace character\n\
 \n\
 @item \\w\n\
 Match any word character\n\
 \n\
 @item \\W\n\
 Match any non-word character\n\
 \n\
 @item \\<\n\
 Match the beginning of a word\n\
 \n\
 @item \\>\n\
 Match the end of a word\n\
 \n\
-@item \\s\n\
-Match any whitespace character\n\
-\n\
-@item \\S\n\
-Match any non-whitespace character\n\
-\n\
-@item \\d\n\
-Match any digit\n\
-\n\
-@item \\D\n\
-Match any non-digit\n\
+@item \\B\n\
+Match within a word\n\
 @end table\n\
 \n\
 Implementation Note: For compatibility with @sc{matlab}, ordinary escape\n\
 sequences (e.g., \"\\n\" => newline) are processed in @var{pat}\n\
 regardless of whether @var{pat} has been defined within single quotes.  Use\n\
 a second backslash to stop interpolation of the escape sequence (e.g.,\n\
 \"\\\\n\") or use the @code{regexptranslate} function.\n\
 \n\
